### ISSUE TITLE
npm install fail

### ISSUE THREAD

**Issue by r0one** [simde-utc/portail#275]
npm install ends with:
```
make: *** [binding.target.mk:129: Release/obj.target/binding/src/binding.o] Error 1
make : on quitte le répertoire « /home/Documents/dev/portail/node_modules/node-sass/build »
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/home/Documents/dev/portail/node_modules/node-gyp/lib/build.js:262:23)
gyp ERR! stack     at ChildProcess.emit (events.js:209:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:272:12)
gyp ERR! System Linux 4.19.0-6-amd64
gyp ERR! command "/usr/bin/node" "/home/Documents/dev/portail/node_modules/node-gyp/bin/node-gyp.js" "rebuild" "--verbose" "--libsass_ext=" "--libsass_cflags=" "--libsass_ldflags=" "--libsass_library="
gyp ERR! cwd /home/Documents/dev/portail/node_modules/node-sass
gyp ERR! node -v v12.10.0
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok 
```
Trying a `npm run dev` despite that gives the following error:
```
Error: Cannot find module '/home/Documents/dev/portail/node_modules/laravel-mix/setup/webpack.config.js'
Require stack:
- /home/Documents/dev/portail/node_modules/webpack/bin/convert-argv.js
- /home/Documents/dev/portail/node_modules/webpack/bin/webpack.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:772:15)
    at Function.Module._load (internal/modules/cjs/loader.js:677:27)
    at Module.require (internal/modules/cjs/loader.js:830:19)
    at require (internal/modules/cjs/helpers.js:68:18)
    at requireConfig (/home/Documents/dev/portail/node_modules/webpack/bin/convert-argv.js:97:18)
    at /home/Documents/dev/portail/node_modules/webpack/bin/convert-argv.js:104:17
    at Array.forEach (<anonymous>)
    at module.exports (/home/Documents/dev/portail/node_modules/webpack/bin/convert-argv.js:102:15)
    at /home/Documents/dev/portail/node_modules/webpack/bin/webpack.js:171:41
    at Object.Yargs.self.parse (/home/Documents/dev/portail/node_modules/webpack/node_modules/yargs/yargs.js:533:18) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/Documents/dev/portail/node_modules/webpack/bin/convert-argv.js',
    '/home/Documents/dev/portail/node_modules/webpack/bin/webpack.js'
  ]
}
```
Probably a problem with webpack. @NastuzziSamy were you able to `npm install` on a fresh install?
**npm v 6.11.3**
**node v 12.0**

**Comment by r0one** [comment]
Maybe the very beginning of the log is important:
```
> node-sass@4.11.0 install /home/Documents/dev/portail/node_modules/node-sass
> node scripts/install.js

Downloading binary from https://github.com/sass/node-sass/releases/download/v4.11.0/linux-x64-72_binding.node
Cannot download "https://github.com/sass/node-sass/releases/download/v4.11.0/linux-x64-72_binding.node": 

HTTP error 404 Not Found
```

**Comment by r0one** [comment]
Apparently it's an environment conf issue since the CI passed.

**Comment by MercierCorentin** [comment]
@CdRom1 I guess you have to downgrade to node 11 as it's written here https://stackoverflow.com/questions/55892917/ionic-build-fails-while-creating-the-project

**Comment by MercierCorentin** [comment]
@CdRom1 CI currently runs with npm@6.11.3. I'll update the documentation.
I have node v10.15.3 installed on my computer and ran a fresh install with it.

**Comment by r0one** [comment]
Is it useful to keep an old version of ionic just because of libsaas (stylesheet preprocessor)?

**Comment by MercierCorentin** [comment]
We actually have to re-think the whole dependencies tree..

**Comment by MercierCorentin** [comment]
Fixed by #323
